scaffold/template/custom/android/fastlane/Fastfile

```diff
@@ -32,20 +32,22 @@ platform(:android) do
 
   desc('Submit a new Internal Build')
   lane(:internal) do
+    upload = true
     begin
       internal_files = google_play_track_version_codes(track:"internal")
-
-      internal_files.length > 0
-
+    rescue => ex
```

Comment:
When does `rescue => ex` code get executed? Basically, how and when will `upload = false` be executed?

Reply:
this gets executed when theres nothing uploaded and code throws exception